support/ci/build_component.ps1

```diff
@@ -17,8 +17,7 @@ $env:HAB_CACHE_KEY_PATH="$job_temp_root/keys"
 Write-Host "--- :key: Generating fake origin key"
 hab origin key generate
 Write-Host "--- :hab: Running hab pkg build for $Component"
-# REMOVE -R AFTER 0.75.0
-hab pkg build -D -R components/$Component
+hab pkg build -D components/$Component
```

Comment:
It looks like this happened in the interim, so we can leave this change out after rebasing.